packages/cloudflare/src/handler.ts

```diff
@@ -47,6 +48,31 @@ export function withSentry<Env = unknown, QueueHandlerMessage = unknown, CfHostM
       markAsInstrumented(handler.fetch);
     }
 
+    /* hono does not reach the catch block of the fetch handler and captureException needs to be called in the hono errorHandler */
+    if (
+      'onError' in handler &&
+      'errorHandler' in handler &&
+      typeof handler.errorHandler === 'function' &&
```

Comment:
So this is always defined right? We don't need to create one for a user if it's not defined?

Reply:
yes, hono has a default error handler: https://github.com/honojs/hono/blob/bb7afaccfd5b6b514da356e069f27eb5ccfc0e3b/src/hono-base.ts#L36